### ISSUE TITLE
Linux 64-bit beta does not provide a check update button

### ISSUE THREAD

**Issue by ggllioio** [AdguardTeam/AdGuardHome#1878]
![QQ截图20200710213038](https://user-images.githubusercontent.com/7471887/87159921-aa1c1f80-c2f4-11ea-86d3-148359199ef4.png)

**Comment by zadigre** [comment]
got the same "problem" on Adguard Home running on my raspberry pi... 
update button was there like a few days ago... but it's not longer there.

**Comment by Eyeborgs** [comment]
There is no button on Windows either

**Comment by ameshkov** [comment]
What exact version you all are running?

**Comment by zadigre** [comment]
v0.102.0-101-gda6c

I believe I got this upgrade last week or the week before. 
I tried to install the beta version today... and still got the same version.

**Comment by ArtemBaskal** [comment]
UI hides check updates button if the response of `/control/version.json` `POST` request has an empty body, which indicates that AdGuardHome is launched with the `--no-check-update` argument. https://github.com/AdguardTeam/AdGuardHome/issues/1726
 Could you check the result of this request?

**Comment by zadigre** [comment]
curl -X POST http://127.0.0.1/control/version.json gives me "forbidden". 
is it the correct command to test?

AdGuard is installed as a service on my raspberry pi... AdGuardHome -s run is shown in ps ax... and confirmed the same command in AdGuardHome.service

**Comment by ameshkov** [comment]
@zadigre just open the browser developer tools and check what response is there

**Comment by zadigre** [comment]
Forbidden is because I was not logged on with my admin password... 
here it is... no sure if this is what you are looking for
![Capture d’écran, le 2020-07-10 à 13 22 01](https://user-images.githubusercontent.com/4810599/87181501-715a5700-c2b0-11ea-8fbb-96fea859ed9d.png)
![Capture d’écran, le 2020-07-10 à 13 22 11](https://user-images.githubusercontent.com/4810599/87181502-715a5700-c2b0-11ea-87ea-05cd1400180d.png)

**Comment by ameshkov** [comment]
Could you please try the version from the "edge" channel?

The list of disros is here:
https://static.adguard.com/adguardhome/edge/version.json

We have not yet announced this new update channel, but it's the closest thing to the master branch you can get.

**Comment by zadigre** [comment]
now running v0.102.0-SNAPSHOT-ab401ca... and I'm getting the same result

**Comment by ameshkov** [comment]
@zadigre could you please check the latest beta or alpha? This issue should've been resolved

**Comment by zadigre** [comment]
@ameshkov 
Still not check update button at the bottom of the page... 
should I try the edge channel too or it will be the same?

**Comment by ameshkov** [comment]
Yeah, it's pretty much the same. So weird, I don't understand why I cannot reproduce this.

This is a client-side issue so I need to know more about your config.

1. OS
2. Browser + version

Just in case, try checking it in the incognito mode.

**Comment by ggllioio** [comment]
The latest v0.103.0-beta1-SNAPSHOT-b9fca8d still has the same problem.
1，Linux OpenWrt 4.19.122 #0 SMP Fri Jun 19 02:58:48 2020 x86_64 GNU/Linux
2，Google Chrome 已是最新版本 版本 85.0.4183.16（正式版本）dev （64 位）
![QQ截图20200714205427](https://user-images.githubusercontent.com/7471887/87428525-093fa400-c615-11ea-9845-a48e2bab74af.png)
![QQ截图20200714205616](https://user-images.githubusercontent.com/7471887/87428535-0e045800-c615-11ea-9740-3b274cf3c410.png)

**Comment by zadigre** [comment]
My main computer is a Mac mini, running the latest stable version of macOS Catalina. 
I'm using Safari 13.1.1. 
But I also tried with Firefox 78.0.2 on my Mac with the same results. 
Last time I tried, I also had same problem with Firefox on a Windows 10 pc. 
Just for this test, I installed Chrome on my Mac... same result.

I have the AdGuard extension on Safari. 
No extension in Firefox or Chrome.

All "protection" are disabled in my router (except the basic firewall obviously).

Besides Adguard Home, I have Unbound that act as a recursive DNS... Adguard goes through unbound to resolve names. 
For the test, I changed my upstream DNS to use Google's servers (bypassing unbound)... same result.

incognito doesn't change anything. 

fyi when login on Adguard this morning, I had a prompt at the top saying 103 beta 1 was available. I click update and it failed. 
I had to manually download and install from the command line.

**Comment by ggllioio** [comment]
This is a screenshot on edge and Firefox.All in privacy mode。

![QQ截图20200714222737](https://user-images.githubusercontent.com/7471887/87437974-912bab00-c621-11ea-9a2c-0e7e3c120c64.png)

**Comment by szolin** [comment]
Thank you!  We've managed to reproduce it on our side and we'll fix it shortly.

**Comment by zadigre** [comment]
@szolin 
should we update to edge channel to test it out as soon as it's available or beta will also be updated shortly?

**Comment by Ulmisch** [comment]
fyi, still the same on Version: 0.103.0-beta1.
No update button...

**Comment by zadigre** [comment]
@Ulmisch 

see @szolin's  message... it will be available soon. 
Fix has already been implemented in the code... I guess we need to wait for the compiled version.

**Comment by ameshkov** [comment]
Please check the latest edge-channel build

**Comment by ggllioio** [comment]
Thank you for your work. The buttons are normal in v0.103.0-beta1-SNAPSHOT-95f4128.

**Comment by zadigre** [comment]
> Please check the latest edge-channel build

I confirm... the check update button is back for me too.

**Comment by zadigre** [comment]
@ameshkov 

I have an other problem with this snapshot that was not present in the beta or previous stable builds... 
the query log on safari is not just empty.... the page is completely blank. 
I tried to clean my temp files and cookies... same result. 
incognito mode... same result. 
it works fine on the same computer with firefox and chrome. 
I tried on my iPhone and iPad in safari (both running iOS 14 beta)... and all is fine.

Should I open an issue for that or you have something quick for me to try?
